src/permuter.py

```diff
@@ -100,12 +100,8 @@ def __init__(
             fns = _find_fns(source)
             if len(fns) == 0:
                 raise Exception(f"{self.source_file} does not contain any function!")
-            if len(fns) > 1:
-                raise Exception(
-                    f"{self.source_file} must contain only one function, "
-                    "or have a function.txt next to it with a function name."
-                )
-            self.fn_name = fns[0]
+            self.fn_name = fns[-1]
+            print("selecting function: ", self.fn_name)
```

Comment:
```suggestion
            print("Selecting function: ", self.fn_name)
```
(would be nicer to have print() only happen in main.py, but we're already inconsistent about that)